packages/koa-shopify-auth/src/auth/client/itp-helper.ts

```diff
@@ -0,0 +1,43 @@
+// Copied from https://github.com/Shopify/shopify_app
```

Comment:
It's probably fine to leave this as is (especially because it is copied from somewhere), but if we felt like making it a bit more readable we could actually have this as  TypeScript with it's own tsconfig that compiles it for direct script tag use if we wanted. Maybe a good future improvement.